modules/recipe-page/views/index.html

```diff
@@ -0,0 +1,1633 @@
+{% extends "@apostrophecms/template:outerLayoutBase.html" %} 
+{% block extraHead %}
+  <link rel="stylesheet" href="/index.css" />
+{% endblock %}
+{% block main %}
+  <div id="journal" class="body template-blog template- show-promo-bar" style="zoom:1">
+    <div id="shopify-section-header" class="shopify-section">
+      <header
+        class="main-header"
+        role="banner"
+        data-section-id="header"
+        data-section-type="header"
+      >
+        <section class="promo-bar main-header__promo-bar promo-bar--active">
+          <a class="promo-bar__title" href="/collections/shop">
+            Free UK shipping on all orders over £250 </a
+          ><button class="promo-bar__close">
+            <span class="screenreader-text">Close Promotional Bar</span>
+          </button>
+        </section>
+
+        <div class="section main-header__inner">
+          <div class="main-header__menu main-header-menu">
+            <button class="main-header-menu__burger burger">
+              <span class="screenreader-text">Open menu</span>
+              <div></div>
+              <div></div>
+              <div></div>
+            </button>
+          </div>
+
+          <nav class="main-nav main-header__nav" role="navigation" id="main-nav">
+            <button class="main-nav__close">
+              <span class="screenreader-text">Close Menu</span>
+            </button>
+            <ul class="main-nav__list">
+              <li class="main-nav__list-item ">
+                <a href="/collections/shop" class="main-nav__list-link">
+                  Shop
+                </a>
+              </li>
+
+              <li class="main-nav__list-item main-nav__list-item--active">
+                <a href="/blogs/journal" class="main-nav__list-link">
+                  Journal
+                </a>
+              </li>
+
+              <li class="main-nav__list-item ">
+                <a href="/pages/our-story" class="main-nav__list-link">
+                  Our Story
+                </a>
+              </li>
+
+              <li class="main-nav__list-item ">
+                <a
+                  href="/blogs/journal/tagged/recipes"
```

Comment:
There may be a plan for this, but it seems bad to be dropping in all these broken links.